web/src/scripts/webflow/licenseSync.mjs

```diff
@@ -205,8 +205,42 @@ const createNewLicenses = async () => {
   console.info(`Created a total of ${newLicenses.length} licenses`);
 };
 
+const getUnUsedLicenses = async () => {
+
+  let localLicensesTemp = loadAllLicenses();
+  localLicensesTemp = localLicensesTemp.filter((it) => it.webflowId === undefined);
```

Comment:
So in the function `getNewLicenses` which is used in createLicenses we see similar logic here removing webflow items that don't have webfowId's from consideration here. 

I assume thats because they aren't tracked. I'm wondering why we have licenses that aren't tracked? 

Also the update section follows slightly different logic only pulling from a subset of our MD content rather than doing this filter above, I'm curious as to whether there is a concrete reason for this or not. 

As I expected when running preview now there are no files to be deleted which is good verification for me.

Reply:
Okay looked into this slightly further and realized that items without webflow ID's are items that haven't been created yet and thus shouldn't be deleted because they don't exist in webflow.